lib/libc/minimal/source/string/strncasecmp.c

```diff
@@ -1,87 +1,28 @@
 /*
- * Copyright (c) 1987 Regents of the University of California.
- * All rights reserved.
+ * Copyright (c) 2018 Intel Corporation
  *
- * Redistribution and use in source and binary forms are permitted
- * provided that this notice is preserved and that due credit is given
- * to the University of California at Berkeley. The name of the University
- * may not be used to endorse or promote products derived from this
- * software without specific written prior permission. This software
- * is provided ``as is'' without express or implied warranty.
+ * SPDX-License-Identifier: Apache-2.0
  */
 
-/*
- * @deftypefn Supplemental int strncasecmp (const char *@var{s1}, const char *@var{s2})
- *
- * A case-insensitive @code{strncmp}.
- *
- * @end deftypefn
- */
```

Comment:
The documentation might be good to be preserved or improved.

Reply:
It's specified by the ISO C standard (and probably POSIX and SuS and XPG and iso C++).  I don't think we really need to be trying to maintain documentation on stuff everyone can look up on every other system.   As long as we comply with the relevant standards, which this seems to (AFAICT there's no requirement for I18N compliance, the docs just say the results might be "different" in different locales).

No one ever calls this in a context where they aren't using ASCII anyway.